homeassistant/components/light/mqtt.py

```diff
@@ -409,7 +419,8 @@ def async_turn_on(self, **kwargs):
         if ATTR_BRIGHTNESS in kwargs and \
            self._topic[CONF_BRIGHTNESS_COMMAND_TOPIC] is not None:
             percent_bright = float(kwargs[ATTR_BRIGHTNESS]) / 255
-            device_brightness = int(percent_bright * self._brightness_scale)
+            device_brightness = int((self._brightness_scale_inverted and 1.0
+                                     - percent_bright or percent_bright) * self._brightness_scale)
```

Comment:
Oh, come on, seriously? I called `autopep8 --in-place --aggressive --aggressive` on that file, isn't that supposed to fulfill all criteria?